ad-ldap-enum.py

```diff
@@ -384,10 +384,7 @@ def get_membership_with_ranges(ldap_client, base_dn, group_dn):
        processing is needed to get the full membership."""
     output_array = []
 
-    # RFC 4515 sanitation.
```

Comment:
@sleventyeleven is there a reason that the RFC 4514 character substitution was removed?

Reply:
No that wasn't intentional, it looks like maybe my versioning was behind here before editing and the pull.